crates/steel-core/src/scheme/stdlib.scm

```diff
@@ -90,13 +90,16 @@
     [(quasiquote (#%unquote x)) x]
 
     [(quasiquote ((#%unquote-splicing x))) (append x '())]
+    [(quasiquote #((#%unquote-splicing x))) (list->vector (append x '()))]
```

Comment:
This is a very blunt and inefficient approach. Not sure if the vector builtins are/could be made available.

Reply:
We probably could - I think they should be available 

That being said quasi quote is some of the most mind bending stuff to work on, so I'm fine with this for now and we can come back to it at some point in the future